### PR TITLE
Corrections to failing code examples

### PR DESCRIPTION
**Pull request by MarcinMoskala**

### CODE REVIEW

File: docs/topics/ksp/ksp-examples.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
+fun KSClassDeclaration.getDeclaredFunctions(): Sequence<KSFunctionDeclaration> =
     declarations.filterIsInstance<KSFunctionDeclaration>()
 ```
 
 ## Check whether a class or function is local
 
 ```kotlin
 fun KSDeclaration.isLocal(): Boolean =
-    parentDeclaration != null && parentDeclaration !is KSClassDeclaration
+    parentDeclaration is KSClassDeclaration
```
Comment (neetopia) [inline docs/topics/ksp/ksp-examples.md:14]: This is not correct. `isLocal()` means if a declaration is within a function body, as per the above comment says, therefore there is no need to fix this.

**Comment by koshachy** [comment]
Hey @MarcinMoskala @neetopia 
Should we proceed with this PR?

**Comment by neetopia** [comment]
The comment I gave was not addressed. I can follow up with a commit to address this comment if the author does not reply.

**Comment by koshachy** [comment]
Hey everyone,
Thanks for contributing to the project.
We created a separate PR for these changes #3806.

**Pull request closed without merging**